src/CommonLib/Processors/ACLProcessor.cs

```diff
@@ -244,365 +249,444 @@ public IEnumerable<string> GetInheritedAceHashes(byte[] ntSecurityDescriptor, st
         /// <param name="hasLaps"></param>
         /// <returns></returns>
         public async IAsyncEnumerable<ACE> ProcessACL(byte[] ntSecurityDescriptor, string objectDomain,
-            Label objectType,
-            bool hasLaps, string objectName = "") {
+           Label objectType, bool hasLaps, string objectName = "")
+        {
+            var (aces, _, _) = await ProcessACL(ntSecurityDescriptor, objectDomain, objectType, hasLaps, objectName, true);
+            foreach (var ace in aces)
+            {
+                yield return ace;
+            }
+        }
+
+        public async Task<(ACE[], bool, bool)> ProcessACL(byte[] ntSecurityDescriptor, string objectDomain,
```

Comment:
Its actually not going to build on CI either

Reply:
Thanks for the review, Rohan!

> Can we move the optional parameter to a non-optional one? Currently, the overload is actually only used if you manually specify the bool, since the other ProcessACL function has the same number of arguments. Let's make it explicit what we're asking for

I made `checkForOwnerRights` optional because `objectName` was already optional in the latest v4, and I couldn't add a required parameter after an optional one. Would you like me to make `objectName` required as well or split it into two methods, one for each signature?

> Its actually not going to build on CI either

What do I need to do to address this?